cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -57,16 +57,26 @@ public class InteractiveConfig {
     private Level logLevel;
 
     public static InteractiveConfig getDefault() {
+        String controllerURI = "localhost:9090";
+        boolean tlsEnabled = false;
+        //Default tls based configurations for pravega cli
+        if (System.getenv("PRAVEGA_CONTROLLER_URI") != null) {
+            controllerURI = System.getenv("PRAVEGA_CONTROLLER_URI");
+            if (controllerURI.startsWith("tls://")) {
+                tlsEnabled = true;
```

Comment:
TLS is being automatically set to true, which is correct however the current code base requires the 'truststore-location` configuration to be specified if this is set to `true`.  To be compatible, the `truststore-location` should be made optional.

Reply:
Currently it is picking up the default JVM trust store if the truststore-location is null or empty

Reply:
I'm sure when I first tested it I needed to pass a trust store, but I must have been mistaken.  You are right, it ignores any truststore set to null or blank.